src/Raven.Server/Web/Studio/StudioCollectionsHandler.cs

```diff
@@ -161,10 +160,10 @@ private void WriteDocument(BlittableJsonTextWriter writer, JsonOperationContext
                                 writer.WriteValue(prop.Token & BlittableJsonReaderBase.TypesMask, prop.Value);
                                 break;
                             case ValueWriteStrategy.SubstituteWithArrayStub:
-                                arraysStubs.Add(prop.Name);
+                                arraysStubs.Add(prop.Name, (prop.Value as BlittableJsonReaderArray).Length);
                                 break;
                             case ValueWriteStrategy.SubstituteWithObjectStub:
-                                objectsStubs.Add(prop.Name);
+                                objectsStubs.Add(prop.Name, (prop.Value as BlittableJsonReaderObject).Count);
```

Comment:
`prop.Value as BlittableJsonReaderObject` suggests that we are expecting that this might be null? If yes, then we can get NRE here, but I think this cannot be null so I would suggest using non-safe cast.

Reply:
ok